components/app/footer/Footer.tsx

```diff
@@ -0,0 +1,96 @@
+// Copyright 2020-2022 SubQuery Pte Ltd authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { Button, Typography } from '../../common';
+import * as React from 'react';
+import styles from './Footer.module.css';
+interface LinksProps {
+  link: string;
+  icon?: string;
+  alt?: string;
+}
+
+const links: LinksProps[] = [
+  { link: 'mailto:hello@subquery.network', icon: '/icons/icon_1.svg' },
+  { link: 'https://twitter.com/subquerynetwork', icon: '/icons/icon_3.svg' },
+  { link: 'https://medium.com/@subquery', icon: '/icons/icon_4.svg' },
+  { link: 'https://github.com/OnFinality-io/subql', icon: '/icons/icon_5.svg' },
+  { link: 'https://youtube.com/c/SubQueryNetwork', icon: '/icons/icon_6.svg' },
+  { link: 'https://t.me/subquerynetwork', icon: '/icons/icon_7.svg' },
+  { link: 'https://www.linkedin.com/company/subquery', icon: '/icons/icon_8.svg' },
+];
+
+export const IconLinks: React.FC = () => (
+  <div className={styles.icons}>
+    {links.map(({ link, icon, alt }) => (
+      <a href={link} key={alt} target="_blank" rel="noreferrer">
```

Comment:
```suggestion
      <a href={link} key={link} target="_blank" rel="noreferrer">
```

It seems the `alt` is optional at `LinksProps` , so it would be good to use `link` as key which is unique and defined